integration-tests/shiro/src/main/resources/application.properties

```diff
@@ -0,0 +1,18 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+
+quarkus.native.additional-build-args = -H:IncludeResources=securityConfig.ini
```

Comment:
For the sake of sharing tips, it reminds me [this](https://camel.apache.org/camel-quarkus/latest/user-guide/native-mode.html#embedding-resource-in-native-executable)

Reply:
I'll try to use suggested way of including resources (I was thinking about it several days ago, but because I didn't need any regex, I kept this simple approach - taken from some quarkus doc)

Reply:
I've refactored PR to use `include-patterns`